nixos/options.tt

```diff
@@ -243,7 +243,7 @@ function ppNix(indent, v) {
   }
 
   if (typeof v == "string") {
-    v = v.replace('<', '&lt;').replace('>', '&gt;');
+    v = v;
```

Comment:
I guess `v = v` is redundant?

Reply:
You're right, I removed the same pattern from both location (and tested both cases) but didn't actually look at what was left at the second location. It's now fixed.